test/unittests/iotools/test_bids_utils.py

```diff
@@ -122,9 +128,35 @@ def test_write_modality_agnostic_files(tmp_path):
 
     from clinica.iotools.bids_utils import write_modality_agnostic_files
 
+    data_dict = {"link": "", "desc": ""}
     assert len(os.listdir(tmp_path)) == 0
-    write_modality_agnostic_files("ADNI", tmp_path)
+    write_modality_agnostic_files("ADNI", data_dict, tmp_path)
     files = os.listdir(tmp_path)
     assert len(files) == 4
     for _, v in EXPECTED_MODALITY_AGNOSTIC_FILES.items():
         assert v in files
+
+
+@pytest.mark.parametrize("study_name", [""])
+@pytest.mark.parametrize("bids_version", ["1.7.0"])
+def test_write_bids_readme(
+    tmp_path,
+    study_name,
+    bids_version,
+    expected_readme_content,
+):
+    """Test function `_write_bids_readme`.
+
+    .. note::
+        Tested independantly for convenience since it takes
+        a different set of input parameters.
+
+    """
+    from clinica.iotools.bids_utils import _write_readme
+
+    data_dict = {"link": "", "desc": ""}
```

Comment:
I don't think testing empty values is right. Since we need to ensure the templating of the readme works, we need to put distinctive values for the fields.